automatic/bulk-crap-uninstaller/tools/chocolateyInstall.ps1

```diff
@@ -1,16 +1,14 @@
 ﻿$ErrorActionPreference = 'Stop'
 
+$toolsPath      = Split-Path $MyInvocation.MyCommand.Definition
+
 $packageArgs = @{
-  packageName            = 'bulk-crap-uninstaller'
-  fileType               = 'exe'
-  url                    = 'https://github.com/Klocman/Bulk-Crap-Uninstaller/releases/download/v3.17/BCUninstaller_3.17_setup.exe'
-  url64bit               = 'https://github.com/Klocman/Bulk-Crap-Uninstaller/releases/download/v3.17/BCUninstaller_3.17_setup.exe'
-  checksum               = '589737f7b5e70558c8e9a13aeab0e654a8c73c4d35ec1d620371665e89b6433d'
-  checksum64             = '589737f7b5e70558c8e9a13aeab0e654a8c73c4d35ec1d620371665e89b6433d'
-  checksumType           = 'sha256'
-  checksumType64         = 'sha256'
-  silentArgs             = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART'
-  validExitCodes         = @(0)
-  softwareName           = 'Bulk Crap Uninstaller'
+  packageName    = 'bulk-crap-uninstaller'
+  fileType       = 'exe'
+  file           = gi $toolsPath\*.exe
+  file64         = gi $toolsPath\*.exe
```

Comment:
Please remove the file64 property, it isn't needed.

Reply:
Why not ?

Reply:
Because there is only 1 installer

Reply:
We have that on number of packages. The point here is that correct installer message will be printed since installers contain both x32 and x64 versions.

Reply:
Talking about  `"Installing 64-bit bulk-crap-uninstaller..."`

Reply:
So I take it you feel it's wrong when it just outputs `"Installing bulk-crap-uninstaller..."`?

The architecture is only outputted when both `file` and `file64` is specified, otherwise they are left out completely (same thing with `url` and `url64`)

Reply:
It also kinda conflicts with [1.1.6](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#116-support-multiple-architectures) (although that specifically mentions urls, but it should be extended to use embedded files as well)

Reply:
OK